safegraph/DETAILS.md

```diff
@@ -2,7 +2,7 @@
 
 We import census block group-level raw mobility indicators from Safegraph,
 calculate functions of the raw data, and then aggregate the data to the
-county and state levels.  MSA and HRR not yet implemented.
+county and state levels.
```

Comment:
Better change to "aggregate the data to the county, MSA, HRR, state, HHS and national levels."

Reply:
thanks, updated